README.md

```
@@ -1,55 +1,31 @@
-   cd webway
-   ```
-
-2. **Install dependencies:**
-   ```bash
-   yarn install
-   ```
-
-## 🔧 Development
-
-### Available Scripts
+### [@void-ark/rust-base](./packages/rust-base)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)